src/shared/infra/http/controllers/transaction/transaction.controller.ts

```diff
@@ -29,4 +42,16 @@ export class TransactionController {
   ): Promise<Transaction> {
     return this.registerUseCase.execute(wallet_id, data);
   }
+
+  @Delete(':id')
+  @HttpCode(204)
```

Comment:
É a primeira vez que estamos utilizando os HttpCode nas rotas, né?

Coloca nas outras rotas se conseguir (e for necessário). 

Ai, utiliza o HttpStatus (acho que é isso):
@HttpCode(HttpStatus.NO_CONTENT)

Reply:
Vou colocar assim, mas não é a primeira rota, o outro soft delete também está desse jeito pra lançar o 204